src/store/index.ts

```diff
@@ -0,0 +1,28 @@
+import { configureStore } from '@reduxjs/toolkit'
+
+export default configureStore({
+    reducer: {
+        list: (state = {todos: []}, action) => {
+            switch (action.type) {
+                case 'ADD_TODO': {
+                    const newState = state;
+                    newState.todos.push(action.payload);
+                    return newState;
+                }
+                case 'REMOVE_TODO': {
+                    return {
+                        ...state,
+                        todos: state.todos.filter((t: any, index: number) => index !== action.payload),
+                    };
+                }
+                case 'CHANGE_TODOS': {
```

Comment:
logic should be changed. When change occurs it should get id and data. then change the content of the todo with that id